superdesk/vocabularies/vocabularies.py

```diff
@@ -520,6 +520,36 @@ def get_field_options(self, field) -> Dict:
         cv = self.find_one(req=None, _id=field)
         return cv and cv.get("field_options") or {}
 
+    def update_vocabulary_from_json(self, vocab_items):
```

Comment:
would be good to have this on base service rather than here so we can reuse this